src/finufft.cpp

```diff
@@ -529,23 +536,23 @@ void FINUFFT_DEFAULT_OPTS(finufft_opts *o)
 // Sphinx sucks the below code block into the web docs, hence keep it clean...
 {
   // sphinx tag (don't remove): @defopts_start
-  o->modeord = 0;
-  o->chkbnds = 1;
-
-  o->debug = 0;
-  o->spread_debug = 0;
-  o->showwarn = 1;
-
-  o->nthreads = 0;
-  o->fftw = FFTW_ESTIMATE;
-  o->spread_sort = 2;
-  o->spread_kerevalmeth = 1;
-  o->spread_kerpad = 1;
-  o->upsampfac = 0.0;
-  o->spread_thread = 0;
-  o->maxbatchsize = 0;
-  o->spread_nthr_atomic = -1;
-  o->spread_max_sp_size = 0;
+  o->modeord = FINUFFT_MODEORD_DEFAULT;
```

Comment:
I find this now too abstract and hard to track down for the user reading the code - also the defaults cannot be viewed in the sphinx docs (note tags). Let's decide about this...